src/main/java/chatbot/TaskList.java

```diff
@@ -0,0 +1,128 @@
+package chatbot;
+
+import java.util.ArrayList;
+import java.util.function.Consumer;
+
+import chatbot.exceptions.InvalidTaskIndexException;
+import chatbot.tasks.Task;
+
+/**
+ * Class that represents the list of tasks the user has created.
+ */
+public class TaskList {
+    public static final int MAX_NUMBER_OF_TASKS = 100;
```

Comment:
Any reason for public?

Reply:
yep should be changed to private static final, thanks!